pandas/core/arraylike.py

```diff
@@ -149,7 +149,7 @@ def __rpow__(self, other):
         return self._arith_method(other, roperator.rpow)
 
 
-def array_ufunc(self, ufunc: Callable, method: str, *inputs: Any, **kwargs: Any):
+def array_ufunc(self, ufunc: np.ufunc, method: str, *inputs: Any, **kwargs: Any):
```

Comment:
np.ufunc currently resolves to `Any`. I would mothball this PR till we have numpy types.

maybe you could just remove the annotation for now so that the ignores can be removed.

or test this with numpy 1.20.0rc2